server/form-pages/utils/questions.ts

```diff
@@ -67,6 +67,16 @@ export const getQuestions = (name: string) => {
           hint: 'For example, British',
         },
       },
+      'pregnancy-information': {
+        isPregnant: {
+          question: `Is ${name} pregnant?`,
+          answers: yesNoOrIDontKnow,
+        },
+        dueDate: {
+          question: 'When is their due date?',
+          hint: 'For example, 27 3 2023',
```

Comment:
we should probably set this as a variable at the top of the function so we have more consistency of hint dates - I think some of them refer to 2007